docs/web-apps/automated-testing/cypress.md

```diff
@@ -66,6 +66,17 @@ values={[
       <th>Supported Browsers</th>
       <th>End of Life</th>
     </tr>
+    <tbody>
+      <tr>
+        <td rowspan='2'>12.6.0</td>
+        <td><b>macOS:</b> 11.00, 12</td>
+        <td rowspan='2'>Chrome, Firefox, Microsoft Edge, Webkit (Experimental)</td>
+        <td rowspan='2'>Mar 01, 2024</td>
```

Comment:
```suggestion
        <td rowspan='2'>March 1, 2024</td>
```

Reply:
Shouldn't we keep those values consistent with the others ?

Reply:
ah, yes - looks like we have not been consistent in not following ap style for abbreviating  months.